discord/gateway.py

```diff
@@ -714,30 +714,30 @@ def __init__(self, *args, **kwargs):
 
     async def initial_connection(self, data):
         state = self._connection
-        state.ssrc = data['ssrc']
-        state.voice_port = data['port']
-        state.endpoint_ip = data['ip']
+        state._VoiceClient__ssrc = data['ssrc']
```

Comment:
instead of always referring to the mangled attributes, why not `state._VoiceClient__x = ssrc = data['ssrc']`

Reply:
Should be resolved where applicable.